src/pipx/main.py

```diff
@@ -27,32 +28,49 @@ def print_version() -> None:
     print(__version__)
 
 
+def indented_wrap(text, subsequent_indent="", split="\n", **kwargs):
+    text = textwrap.dedent(text).strip()
+    width = shutil.get_terminal_size().columns - 2
```

Comment:
Please add `fallback=(80,24)` argument to `get_terminal_size` .  It can fail to return real measurements in edge cases.

Reply:
fallback defaults to `(80, 24)`.

https://docs.python.org/3/library/shutil.html#shutil.get_terminal_size

```python
def get_terminal_size(fallback=(80, 24)):
```

Reply:
ah ok very good.